.github/workflows/react_tests.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: React Tests
+
+on:
+  push:
```

Comment:
Any particular reason for running the tests on `push` to master as well as pull requests?